docs/configs-npb-tests/run_npb.py

```diff
@@ -56,13 +56,18 @@ def writeBenchScript(dir, bench):
     file_name = '{}/run_{}'.format(dir, bench)
     bench_file = open(file_name,"w+")
     bench_file.write('/home/gem5/NPB3.3-OMP/bin/{} \n'.format(bench))
+
+    # sleeping for sometime makes sure
+    # that the benchmark's output has been
+    # printed to the console
+    bench_file.write('sleep 5 \n')
     bench_file.write('m5 exit \n')
     bench_file.close()
     return file_name
 
 if __name__ == "__m5_main__":
     (opts, args) = SimpleOpts.parse_args()
-    kernel, disk, benchmark, num_cpus = args
+    kernel, disk, cpu, benchmark, num_cpus = args
```

Comment:
If we only support kvm and atomic, there should be a call to `m5.fatal` if the CPU isn't correct.

Also, we should stop using SimpleOpts :), but that can be for another day.

Reply:
It will need small changes in run script to add other cpus as well. For now, I added a call to m5.fatal in a new commit for other cpu types